app/views/trips/_form.html.erb

```diff
@@ -8,6 +8,7 @@
     <%= f.input :end_date, order: [:day, :month, :year], selected: 5.days.from_now %>
     <%= f.label :school %>
     <%= f.collection_select :school_id, School.all, :id, :name, include_blank: "" %>
+    <br>
```

Comment:
Elle made a comment on one of my views that I should use css styling rather than setting a break line in the html.

Reply:
Done in `a61a204`!